### PR TITLE
udpate license info

### PR DESCRIPTION
**Pull request by osallou**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pash/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ requirements:
 
 about:
   home: http://www.bioinformatics.bbsrc.ac.uk/projects/bismark/
```
Comment (abretaud) [inline recipes/pash/meta.yaml]: It's the url which is wrong, it should be: https://genboree.org/pash-supp/
=> there, the licence is not specified

Comment (osallou) [inline recipes/pash/meta.yaml]: should contact author, if license is not specified, it is not sure we ware allowed to redistribute software....

Comment (abretaud) [inline recipes/pash/meta.yaml]: Yes, *we* should (you? me? hehe)

Comment (abretaud) [inline recipes/pash/meta.yaml]: I just sent an email to the author

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Pull request closed without merging**